add notes for session var (#82)
risingwavelabs/risingwave-docs: operate/view-configure-runtime-parameters.mdx

For details about the `set_config()` function, see [System administration functions](/sql/functions/sys-admin#set-config), and for details about the `SET` command, see [SET](/sql/commands/sql-set).

<Note>

> `SET` only takes effect in the current session.

Reviewer: ```suggestion
`SET` applies only to the current session.
```

</Note>

You can also use the [ALTER SYSTEM SET](/sql/commands/sql-alter-system) command to set a system-wide default value for a runtime parameter. This configuration will then be applied to every new session.

<Note>

> `ALTER SYSTEM SET` only takes effect for new sessions. The value of the runtime parameter will stay unchanged in the current session.

Reviewer: ```suggestion
`ALTER SYSTEM SET` takes effect in new sessions. The value of the runtime parameter remains unchanged in the current session.
```
